open-api-spec.yaml

```
@@ -1280,6 +1318,44 @@ components:
         - hasNextPage
         - hasPreviousPage
       additionalProperties: false
+    json-web-key:
+      type: object
+      properties:
+        kid:
+          type: string
+        use:
+          type: string
+          enum:
+            - sig
+        kty:
+          type: string
+          enum:
+            - OKP
+        crv:
+          type: string
+          enum:
+            - Ed25519
+        x:
+          type: string
+          format: binary
```

```suggestion
          pattern: "^[a-zA-Z0-9-_]+$"
          description: The base64 url-encoded public key.
```
https://www.rfc-editor.org/rfc/rfc7517#section-3

See `sharedSecret`:
https://github.com/interledger/open-payments/blob/4c873dba89164decffbe84905d12f1d4ec045389/open-api-spec.yaml#L832-L836